contact.html

```diff
@@ -0,0 +1,134 @@
+                        </li>
+                    </ul>
+                    <!-- <form class="d-flex">
+                  <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                  <button class="btn btn-outline-success" type="submit">Search</button>
+                </form> -->
+                </div>
+            </div>
+        </nav>
+    </header>
+    <div class="container py-4">
+        <form id="contactForm">
+            <div class="mb-3">
+                <label class="form-label" for="name">Name</label>
+                <input class="form-control" id="name" type="text" placeholder="Name" />
+            </div> <!-- Email address input -->
+            <div class="mb-3">
+                <label class="form-label" for="emailAddress">Email Address</label>
+                <input class="form-control" id="emailAddress" type="email" placeholder="Email Address" />
+            </div>
+            <div class="mb-3">
+                <label class="form-label" for="message">Message</label>
+                <textarea class="form-control" id="message" type="text" placeholder="Message"
+                    style="height: 10rem;"></textarea>
+            </div>
+            <div class="d-grid">
+                <button class="btn btn-primary btn-lg" type="submit">Submit</button>
+            </div>
+        </form>
+    </div>
+    <p>In Quebec city? Come meet me at my office!</p>
+    <iframe
+    width="100%"
+    height="350"
+    frameborder="0" style="border:0"
+    referrerpolicy="no-referrer-when-downgrade"
+    src="https://www.google.com/maps/embed/v1/place?key=AIzaSyCYaIy-PFdUgFMJoPUkGELFMMtvF3mnjMU&q=SOS+Aventures+:+jeux+d'évasion,+2095+Rue+Frank-Carrel,+Québec+City,+Quebec+G1N+4L8"
+    allowfullscreen>
+  </iframe>
+    <script src="js/bootstrap.min.js"></script>
+    <footer>
+        <div class="p-3 mb-2 bg-dark text-white">
+            <ul class="nav justify-content-center">
+                <li class="nav-item">
+                    <nav class="nav flex-column" style="margin-right: 10px;">
+                        <a href="about.html" class="text-white bg-dark">About</a>
+                        <a href="contact.html" class="text-white bg-dark">Contact</a>
+                        <a href="projects.html" class="text-white bg-dark">Projects</a>
+                    </nav>
+                </li>
+                <li class="nav-item">
+                    <a href="https://www.artstation.com/walldo">
+                        <img src="assets/Artstation.png" style="max-width: 70px; margin: 0px 10px;" alt="artstation" />
+                    </a>
+                </li>
+                <li class="nav-item">
+                    <a href="https://www.linkedin.com/in/charles-%C3%A9tienne-cloutier-904366133/">
+                        <img src="assets/linkedIn.png" style="max-width: 70px; margin: 0px 10px;" alt="linkedin" />
```

Comment:
Like in other pages, your images here are not displayed on the server. This is probably because the server is case-sensitive. So, here it should be `assets/linkedin.png`